### ISSUE TITLE
EN 15804+A2, chapter 7.1, i), variability of results

### ISSUE THREAD

**Issue by RobertSpang** [InDataWG/ILCD-EPD-Data-Format#17]
### I raise this change request because...

- [ ] the ILCD+EPD format lacks compliance with regualtions.
- [X] the ILCD+EPD format lacks conformaty with standards.
- [ ] the ILCD+EPD format needs to be improved regarding existing format or master data elements.
- [ ] there is another problem or use case I´m trying to address.

### Problem description

"in the case where an EPD is declared as an average environmental performance for a number of products a statement to that effect shall be included in the declaration together with a description of the range/ variability of the LCIA results if significant"

### Solution description

Could be included in the following existing fields:
- A1.4 General comment on data set
- B2.2 Use advice for data set

Partly covered by B1.2 "subtype"

### Relevance or added value

Medium relevance

**Comment by okworx** [comment]
Suggestion: use the same systematic as defined in ISO 22057, ideally amalgamate the variants given there for EN15804 and ISO21930 in order to have a unified and universal approach.

So we need an enum for describing this
•	Valid for a single product where variability is not described
•	Valid for a single product where variability is described
•	Valid for a range of products where variability is not described
•	Valid for a range of products where variability is not described
as well as a number for the variation of described.

Add a freetext comment field.

**Comment by okworx** [comment]
Let's consider a significant variation to be one that is larger than 10% (following ISO 14044).
We're looking at multiple indicators and multiple modules, so there is no single variation per se. It would first need to be defined which variation exactly.

**Comment by okworx** [comment]
How is variability defined?
How is "product" defined?

**Comment by okworx** [comment]
@ domain experts: please review and approve

**Comment by RobertSpang** [comment]
so this is a conditional field: if average, then it´s mandatory. if mandatory, lets add the "classes" of ISO 14044 Annex B to identify the variability. this way we can also make it machine readable. 
![Classes ISO 14044 annex B](https://github.com/user-attachments/assets/9bd8e894-8786-4ad6-b77b-a210debf827b)
Ok from your side?

**Comment by 1aun** [comment]
I would advise against it. 

This, to me, is a feature request for the business logic layer of a software application displaying the data.

The data format should, imho, enclose exclusively the data that's strictly necessary -- that's data that is not recoverable, if omitted in the XML of a dataset.
The other aspect here is the fact that attaching the interpretation of the value is not something that's specific to the dataset/XML at hand. It is much more static in that it is probably done in the very same way for any dataset XML.

With this philosophy we keep not only the (already huge) memory footprint low but also ensure maximal flexibility: Anybody can do whatever interpretation of the value.

Our goal should be to _provide/persist just enough data_ to allow any analysis (by experts or software) in a sufficiently normalised way to be efficiently processed.

PS:
Without the ISO 14044 Annex B classes in the XMLs it's still fairly easy for any software company to provide filters that accept these concepts/labels as search criteria. The necessary queries, except for syntax, do not even diverge much from what you've posted in the screenshot.

**Comment by RobertSpang** [comment]
no - variablity of an average cannot be derived from declared values. Variability is determined by the underlying data (= foreground data, recipes etc.). So we could use the classes from ISO 14044 to declare variability:
<=2,5% neglectible variation
>2,5 - 10% minor or "normal" variation
>10% - 25% relevant variation
>25% - 50% significant variation
Makes sense?

**Comment by 1aun** [comment]
Thanks for the quick response!

The suggested implementation would look like this.

```xml
<variability>
   <manufacturerVariability type="Single production site" variation="5"/>

   <productVariability type="Range of products where variability is described" variation="20"/>

   <variabilityDescription xml:lang="en">... desc ...</variabilityDescription>
</variability>
```

I was referring to the `variation` value here, which in both instances holds the numerical value for variation in percent.

I think this covers it without locking us into ISO 14044?

**Comment by RobertSpang** [comment]
This needs explanation; please set up a meeting to explain this to me. thx.

**Comment by szwerenz** [comment]
I also don`t understand the statet solution a meeting would help. Otherwise i don`t see the necessity to have it as a machine readable informaiton. I think a describtion in an existing field should be sufficient.

@EmilieBrisson what do you think, do you need that information on building level?

**Comment by EmilieBrisson** [comment]
I agree (sort of) with @szwerenz. As of right now, we do not use variability. But it might come. 
I do not think the ide above is a bad idea, but maybe with some more explanation for the group here, would be nice. Though I would maybe suggest UUID for the specific identified cases, from the screenshot @RobertSpang did. Othrwise we might get a lot of unusefull entries.

**Comment by okworx** [comment]
Summary from today's meeting: we will include both an option to state a discrete value in percent as well as one to state the range as suggested above, i.e. an optional attribute @variabilityRange is to be added.

**Comment by okworx** [comment]
What we need to do is to define "single product". Is that referring to a class of the same products (i.e. multiple manufacturers) or the same physical one.
